dactyl-config.yml

```diff
@@ -2507,6 +2507,15 @@ pages:
         targets:
             - local
 
+    -   md: use-cases/contribute-code-to-ripple-lib.md
+        html: contribute-code-to-ripple-lib.html
+        funnel: Use Cases
+        template: template-use-case.html
+        useful_background:
+            - consensus.html
+            - amendments.html #TODO add to useful_background: stand-alone mode, rippled.cfg reference (future feature)
```

Comment:
suggested useful background:
- rippleapi-reference.html
- get-started-with-rippleapi-for-javascript.html

Reply:
This is great - thank you for noticing this. I think I just copied the existing `useful_background` values from the `rippled` use case - no bueno. These are much more USEFUL. 🙌